README.md

```diff
@@ -39,15 +44,17 @@ Notice that:
 
 ## Attachments
 
-Inheriting from `TRENZ.Lib.RazorMail.Core.MailTemplateBase<T>` also gives us the convenience methods `AttachFile()` and `InlineFile()`. These differ only in whether an attachment is intended for download, or for inline display.
+Inheriting from `TRENZ.Lib.RazorMail.Core.MailTemplateBase<T>` also gives us the convenience methods `AttachFile()` and
+`InlineFile()`. These differ only in whether an attachment is intended for download, or for inline display.
 
 For example, to show an image inline, you simply do:
 
 ```cshtml
 <img src="@InlineFile("My Company Logo.png")" />
 ```
 
-That's it. This attaches the image as a file, then references it using `cid` format. Because the image is attached, this also doesn't require your users to enable loading external images.
+That's it. This attaches the image as a file, then references it using `cid` format. Because the image is attached, this
```

Comment:
Can we add a link for `cid` to reference a spec where this is defined? I doubt everyone looking at this will know what a `cid` is in this context.

Reply:
Documentation on this is meh, but I've amended the README.